### PR TITLE
Make a start to PEP8 compliant code

### PR DESCRIPTION
**Pull request by basnijholt**
This is just a small start :)

The Java style names still hurt my eyes but changing that will mean breaking compatibility.

### CODE REVIEW

**Comment by johnkgamble** [comment]
This looks great to me, and I don't see any problems. I'm going to wait for Adam to run some functional tests before merging it in just to be sure.

**Comment by basnijholt** [comment]
Ah, I see that now.

Will change that when I get the time.

File: qmt/batchModel.py [hunk 1]
```diff
@@ -406,18 +444,28 @@ def addThomasFermi2dTask(self, region, grid, slice_name, name=None, write_data=T
...
+                'electrostatic_potential',
+                'potential_energy',
+                'effective_mass']
+        else:
             write_data = []
-        if plot_data is True:
-            plot_data = ['density', 'electrostatic_potential', 'potential_energy', 'effective_mass',
-                         'bare_bands']
-        elif plot_data is False:
+        if plot_data:
```
Comment (basnijholt) [inline qmt/batchModel.py]: `plot_data` can be a list or bool

**Comment by adamfrees** [comment]
@basnijholt @johnkgamble - functional tests passed!

**Comment by basnijholt** [comment]
@jgukelberger I've reverted those changes. Sorry for missing it.

**Comment by johnkgamble** [comment]
OK, thanks for the changes @basnijholt ! I'm merging now.

**Pull request merged**